modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -130,6 +136,7 @@ const ColorPicker = ({
   resetColor,
   value = '',
   showCustomHexInput = false,
+  transformOrigin,
```

Comment:
@lychyi - Is this just a style preference. Curious the reason for the addition.

Reply:
We are generally explicit about destructuring out documented props. The term `...elemProps` refers to the element props (or remainder, rest, etc.) spread onto the containing element (think `id`, `aria-*`, etc.). In this case `transformOrigin` would in fact go to the same place but we're explicitly directing it to go to the `Popup` where it belongs.

It sounds like of like an odd style preference, but we've ran into things in the past when this has bitten us and we weren't aware of overriding props or spreading it to the wrong place until we shipped the code.